todo_app/data/todo_items.py

```diff
@@ -0,0 +1,22 @@
+class Item:
+
+    def __init__(self, id, name, status = 'To Do'):
+        self.id = id
+        self.name = name
+        self.status = status
+
+    @classmethod
+    def fromTrelloCard(cls, card, list):
+        return cls(card['id'],card['name'],list['name'])
+
+    def start(self):
+        self.status='Doing'
+
+    def complete(self):
+        self.status='Done'
+
+    def reset(self):
+        self.status= 'To Do'
```

Comment:
I don't think we ever use these methods (nor are we likely to, since we'll tell Trello to move the item and then just load it in the new state) so we can get rid of these